src/generateVehicleCapabilities.ts

```diff
@@ -0,0 +1,59 @@
+import type { DatabaseConfig, VehicleTypeConfig } from "./config";
+import db from "./db";
+
+const DEFAULT_CAPACITY = 78;
```

Comment:
Default capacity shouldn't be hardcoded. It's documented as optional environment variable

DEFAULT_VEHICLE_CAPACITY.

Logic should be "If DEFAULT_VEHICLE_CAPACITY is defined in env, use it. If not, fallback to 78"

Reply:
Actually... Since we're already checking for capacity to exist in config.ts "generateCapabilities" function, this check shouldn't be here at all. Remove this and other related stuff and make sure it still works as intended.